schema.sql

```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
-);
+    id int NOT NULL  PRIMARY KEY,
+	name VARCHAR(100),
+	date_of_birth date NOT NULL,
+	escape_attempts integer,
+	neutered boolean NOT NULL,
+	weight_kg numeric NOT NULL,
+	species VARCHAR(50)
```

Comment:
- [ ] It would be better if we use the `ALTER` procedure to add the `species` variable to our Animals table. Instead of reinventing the wheel. This also helps us to familiarise ourselves with the `ALTER` procedure

Reply:
Alright, done